app/views/layouts/application.html.slim

```diff
@@ -61,11 +61,12 @@ html.govuk-template.app-html-class lang="en"
 
       main.govuk-main-wrapper#main-content role="main"
         - unless flash.empty?
-          .govuk-main-wrapper__notification
-            .govuk-width-container
-              = render "shared/components/notification"
```

Comment:
Now that this `_notification` partial is only used by one file (`publishers/vacancies/documents/destroy.html.slim`), we can update `destroy.html.slim`, and delete the `_notification` partial.